rasa_core/processor.py

```diff
@@ -323,7 +323,7 @@ def should_predict_another_action(action_name, events):
         return not is_listen_action
 
     def _schedule_reminders(self, events: List[Event],
-                            dispatcher: Dispatcher) -> None:
+                            dispatcher: Dispatcher, tracker) -> None:
```

Comment:
please add `tracker` type

Reply:
I had already changed that in the PR. Do I still have to do something here?

Reply:
let's add `tracker: DialogueStateTracker`, also we usually put it before dispatcher, I suggest do it here as well for consistency, so
```
def _schedule_reminders(self,
                        events: List[Event],
                        tracker: DialogueStateTracker,
                        dispatcher: Dispatcher) -> None:
```